tests/test_dataset_common.py

```diff
@@ -135,10 +135,7 @@ def check_load_dataset(self, dataset_name, configs, is_local=False, use_local_du
                     logger.info("Skip tests for this dataset for now")
                     return
 
-                if config is not None:
-                    version = config.version
-                else:
-                    version = dataset_builder.VERSION
+                version = config.version if config else dataset_builder.config.version
```

Comment:
same
```suggestion
                version = dataset_builder.config.version
```

Reply:
@lhoestq I did a refactoring that strictly preserved previous functionality.

Your suggestion changes previous functionality:
- before: we could have configs with different versions and they would be handled properly (even configs with the same name but different versions would be handled properly)
- with your change: we assume all configs have the same version (and we also assume that all configs have the same version as the default config or first BUILDER_CONFIGS

Is this change in behavior intended?

Reply:
Oh ok I see - please ignore my comment then